threescale_api/resources.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, *args, entity_name='limit', entity_collection='limits', metri
         self._metric = metric
 
     @property
-    def metric(self) -> 'Metric':
+    def metric(self):
```

Comment:
why this change?

Reply:
+1

Reply:
There can be Limit linked to Metric linked to Backend. Then it is not Metric, but BackendMetric.

Reply:
@mkudlej ^^